private/bufpkg/bufplugin/bufplugindocker/docker.go

```diff
@@ -182,6 +190,14 @@ func (d *dockerAPIClient) Delete(ctx context.Context, image string) (*DeleteResp
 	return &DeleteResponse{}, nil
 }
 
+func (d *dockerAPIClient) Inspect(ctx context.Context, image string) (*InspectResponse, error) {
+	inspect, _, err := d.cli.ImageInspectWithRaw(ctx, image)
+	if err != nil {
+		return nil, err
+	}
+	return &InspectResponse{ImageID: inspect.ID}, nil
```

Comment:
Doing an inspect here allows us to check for an image ID in the OCI registry no matter if we pass a .zip file or an image with the `-image` flag.